octopusdeploy/resource_feed.go

```diff
@@ -9,10 +9,11 @@ import (
 
 func resourceFeed() *schema.Resource {
 	return &schema.Resource{
-		Create: resourceFeedCreate,
-		Read:   resourceFeedRead,
-		Update: resourceFeedUpdate,
-		Delete: resourceFeedDelete,
+		DeprecationMessage: "use type-specific feed resources instead (ex: octopusdeploy_nuget_feed, etc",
```

Comment:
The plugin is still in beta, I think we can remove things without needing to deprecate them